nanome/_internal/_ui/_button.py

```diff
@@ -203,6 +203,13 @@ def _set_all(self, value):
             self._selected_highlighted = copy.deepcopy(value)
             self._unusable = copy.deepcopy(value)
 
+        def _set_each(self, **values):
+            self._idle = copy.deepcopy(values.get('idle')) or self._idle
+            self._highlighted = copy.deepcopy(values.get('highlighted')) or self._highlighted
+            self._selected = copy.deepcopy(values.get('selected')) or self._selected
+            self._selected_highlighted = copy.deepcopy(values.get('selected_highlighted')) or self._selected or self._selected_highlighted
```

Comment:
`_selected` shouldn't take priority over `_selected_highlighted` here, same

Reply:
ah, yes, you're right

Reply:
also I'm not convinced this definition needs to exist. I think maybe only button.py needs one